src/components/dropdown/dropdown-item.ts

```diff
@@ -25,17 +51,45 @@ class BXDropdownItem extends LitElement {
     if (!this.hasAttribute('role')) {
       this.setAttribute('role', 'option');
     }
+    this._hKeydown = on(this, 'keydown', this._handleKeydown);
```

Comment:
Seems that `@HostListener()` can be used for here.

Reply:
Thought I grabbed that one 😮 oops